inputs/capacity_exogenous/ReEDS_generator_database_final_EIA-NEMS.csv

```diff
@@ -28029,25 +28029,24 @@ o-g-s,p103,218,516.827,2081,2081,2081,2081,2081,2026,TRUE,10623,p26147,St. Clair
-upv,p103,212,18333,2055,2055,2055,2055,2055,2041,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,18333,18333,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
-upv,p103,212,18625,2055,2055,2055,2055,2055,2042,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,18625,18625,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
-upv,p103,212,18621,2055,2055,2055,2055,2055,2043,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,18621,18621,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
-upv,p103,212,18353,2055,2055,2055,2055,2055,2044,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,18353,18353,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
-upv,p103,212,18197,2055,2055,2055,2055,2055,2045,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,18197,18197,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
+upv,p103,212,70,2054,2054,2054,2054,2054,2024,TRUE,,p26121,Muskegon County,MI,43.233,85.993,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,112,112,,,,,2024,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,0,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
+upv,p103,212,290,2055,2055,2055,2055,2055,2025,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,712,712,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
+upv,p103,212,1915,2055,2055,2055,2055,2055,2026,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,"2,877","2,877",,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,

```

Comment:
```suggestion
upv,p103,212,1915,2055,2055,2055,2055,2055,2026,TRUE,,p26121,Muskegon County,MI,43.232797,-85.99345,,,,,MISO,65572,MSP,Muskegon Solar,,,,,,,,,,,,1,2877,2877,,,,,2025,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,upv_n_fsu,n,fsu,1,0,"(U) Under construction, less than or equal to 50 percent complete",,,FALSE,
```
These numbers are formatted as "strings" rather than numbers.